lang/ua.php

```diff
@@ -112,7 +112,13 @@
       ],
       'intro' => [
         0 => 'Кожна грошова пожертва на рахунок нашого фонду допомагає нам досягати наших статутних цілей.',
-        1 => 'Протягом року ви можете зробити пожертву безпосередньо на рахунок Фонду в банку PKO BP<br>номер рахунку: <b>' . $site_account . '</b><br>з назвою: <b>“Пожертва на статутні цілі”</b>'
+        1 => 'Протягом року ви можете зробити пожертву безпосередньо на рахунок Фонду в'
+      ],
+      'bank_details' => [
+        'bank_name' => 'PKO Bank Polski S.A.',
```

Comment:
bank name not so important, more important is address
so add address field to list of account credentials